basictracer/tracer.py

```diff
@@ -64,6 +89,12 @@ def start_span(
             # TODO only the first reference is currently used
             parent_ctx = references[0].referenced_context
 
+        # retrieve the active SpanContext
```

Comment:
are we dropping entirely the old `start_span` behavior (that is ignoring the active span)? I mean with this change current instrumentation will have a different behavior, so we're not deprecating an API, but making an immediate breaking change. Though I know that `start_span` is very appealing :)

Reply:
Yes, I'd say so - from now on, we will be by default inheriting from any active `Span`/`Scope` (unless the `ignore_active_span` parameter is used). In this case, this defacto breaking change is needed - the alternative, which is to keep yet another overload, is not very appealing IHMO.

But this is a good time to mention that in the README and the PR itself ;) Will update the description.

Reply:
I'm totally fine to avoid intermediate methods in our API (`start_manual` wasn't great). On the other hand, I would say that it must be clear we're breaking this API (in the sense of changing the behavior), though our deprecation policies should move towards something more reliable in the future for implementers and developers.

Reply:
Definitely, we should be clear on this. And just as it happened with the Java API, I'd expect this to be one of the very few API breaking changes to happen hopefully ;)